src/raw.rs

```diff
@@ -1,28 +1,16 @@
-// Allow dead code in here in case I want to publish it as a crate at some
-// point.
-#![allow(dead_code)]
-
-extern crate enum_primitive_derive;
-extern crate libc;
-extern crate num_traits;
-
 use std::cmp::Ordering;
 use std::ffi::{c_ulonglong, CString};
 use std::os::raw::{c_char, c_double, c_int, c_long, c_longlong, c_void};
 use std::ptr;
 use std::slice;
 
-use crate::RedisResult;
 use bitflags::bitflags;
 use enum_primitive_derive::Primitive;
 use libc::size_t;
 
 use crate::error::Error;
 pub use crate::redisraw::bindings::*;
-use crate::{context::StrCallArgs, RedisString};
-use crate::{RedisBuffer, RedisError};
-
-const GENERIC_ERROR_MESSAGE: &str = "Generic error.";
+use crate::{context::StrCallArgs, RedisBuffer, RedisError, RedisString, Status};
```

Comment:
Do we need to change all those functions on this file to `public(crate)` or do we count on the fact that we are not exposing this mod as public?

Reply:
I guess this is the answer: https://github.com/RedisLabsModules/redismodule-rs/pull/375#discussion_r1404184390

Reply:
Yes :-)